stats/stats-server/src/server.rs

```diff
@@ -219,6 +220,27 @@ fn init_runtime_setup(
     Ok(Arc::new(setup))
 }
 
+async fn check_if_unsupported_charts_are_enabled(
+    setup: &RuntimeSetup,
+    blockscout_db: &DatabaseConnection,
+) -> anyhow::Result<()> {
+    let migrations = BlockscoutMigrations::query_from_db(blockscout_db).await?;
+    if !migrations.denormalization {
+        let mut all_enabled_charts_with_deps = setup.update_groups.values().flat_map(|g| {
+            g.group
+                .enabled_members_with_deps(&g.enabled_members)
+                .into_iter()
+        });
+        if all_enabled_charts_with_deps.any(|key| key.name() == NewBuilderAccounts::name()) {
+            return Err(anyhow!(
+                "Builder accounts are not supported without denormalized database. \
+                Ensure denormalization is complete or disable the corresponding charts."
+            ));
+        }
+    }
+    Ok(())
```

Comment:
maybe like (just wrote in github, didnt check it with compiler): 
```suggestion
    if !migrations.denormalization {
        let charts_without_normalization = &[NewBuilderAccounts::name()];
        let mut all_enabled_charts_with_deps = setup.update_groups.values().flat_map(|g| {
            g.group
                .enabled_members_with_deps(&g.enabled_members)
                .into_iter()
        });
        if Some(key) = all_enabled_charts_with_deps.find(|key|  charts_without_normalization.contains(key.name())) {
            return Err(anyhow!(
                "chart with name '{key}' is not supported without denormalized database. \
                Ensure denormalization is complete or disable the corresponding charts."
            ));
        }
    }
    Ok(())
```